chapter01/dxl/src/main/java/com/spring/services/HelloWorld.java

```diff
@@ -0,0 +1,29 @@
+package com.spring.services;
+
+
+public class HelloWorld implements EchoService {
+
+	private String message = "Hello, world!";
+
+	/*
+	 * I initially forgot this, but a bean needs a constructor with no arguments,
+	 * otherwise you get a runtime exception when initialising the object.
+	 */
+	public HelloWorld () {
+		super();
```

Comment:
this isn't necessary because every object implicitly contains an empty constructor.  Also since this class doesn't have an extends this is calling the constructor in Object.  I don't see a reason to do this.